### ISSUE TITLE
Improvement for more distinctive colors  in the Test navigation's highlighted result colors (successful or failed) for accessibility for color blind users

### ISSUE THREAD

**Issue by NickGraham5** [SeleniumHQ/selenium-ide#1023]
## 🚀 Feature Proposal

- Type: **Low Priority**
- Assumption on user base size for this request: **Low**

- Improvement on color indicators for when a test passes or fails a run in the "Test" navigation menu
- Making the color more distinctive to users with red-green color blindness

![IDE](https://user-images.githubusercontent.com/43630339/80029630-aecaf780-84ac-11ea-877c-44655d1d678b.png)

**Proposal #1**
- Improve the contrast between the red & green colors so there is a clear difference between the colors to improve accessibility for color blind users. 

**Proposal #2**
- Allow users to have limited change access to specific alert colors that are being utilized and displayed in the IDE.
-- Allow the user change alert colors with success or failure color to blue or another option.

**Proposal #3**
- Allow users to configure all the colors that are being utilized and displayed in the IDE.

## Motivation
- I just started using this tool and I can already tell it is going to help improve our current process with testing. I'm excited to get my hands dirty with the tool and dive in. As I was setting up and running test, I found the colors for the red (fail) and green (pass) to be very difficult to see. The colors more or less blend together for me and probably a small portion of the user base. I'm color blind and i'm pretty much used trying to find work around applications with colors, but for this specific application it is important to me that I don't miss a failed test due to colors. I know the audience for this request is small, but it could be used to improve the product in others ways as well.

## Example

**Proposal #1**
- Basic improvement for accessibility to color blind users trying to indicate a pass or fail.

**Proposal #2** 
- Improvement to allow specific alert colors to be changed with limited selection of specified colors.

**Proposal #3**
- Full configuration of all colors being utilized in the extension for any available static or hex color.
- Future potential use for proposal #3 could be used for a future feature with allow users to set specific color highlighting for specific test run errors that might be encountered.
